src/Rings/MPolyQuo.jl

```diff
@@ -365,6 +357,114 @@ function Oscar.addeq!(a::MPolyQuoElem, b::MPolyQuoElem)
   return a
 end
 
+@doc Markdown.doc"""
+    simplify(a::MPolyQuoIdeal)
+Simplify the ideal `a` w.r.t. the parent quotient ring. Replace inplace `a.SI`to the unique simplified representation.
+
+CAVEAT: The implementation proceeds by computing a Groebner basis of the quotient ideal first. This may take some time.
+
+# Examples
+```jldoctest
+julia> R, (x, y) = PolynomialRing(QQ, ["x", "y"], ordering=:degrevlex)
+(Multivariate Polynomial Ring in x, y over Rational Field, fmpq_mpoly[x, y])
+
+julia> Q, q = quo(R, ideal(R, [x^3*y^2-y^3*x^2, x*y^4-x*y^2]))
+(Quotient of Multivariate Polynomial Ring in x, y over Rational Field by ideal generated by: x^3*y^2 - x^2*y^3, x*y^4 - x*y^2, Map from
+Multivariate Polynomial Ring in x, y over Rational Field to Q defined by a julia-function with inverse
+)
+
+julia> I = ideal(Q,[x^3*y^4-x+y, x*y+y^2*x])
+MPolyQuoIdeal(x^3*y^4 - x + y, x*y^2 + x*y)
+
+julia> simplify(I)
+Singular Ideal over Singular Polynomial Quotient Ring (QQ),(x,y),(dp(2),C) with generators (x^2*y^3 - x + y, x*y^2 + x*y)
+```
+"""
+function simplify(a::MPolyQuoIdeal)
+  R = base_ring(a)
+  J = R.I
+  groebner_assure(J)
+  singular_assure(J.gb)
+  oscar_assure(a)
+  singular_assure(a.I)
+  red   = reduce(a.I.gens.S, J.gb.S)
+	SR		=	singular_ring(R)
+	a.SI	=	Singular.Ideal(SR, gens(red))
+	return a.SI
```

Comment:
This function is changing `a`, but returning `a.SI`?

I think we do not want to expose `sideal` to the user.